### PR TITLE
[s] I give up and rename it to what it actually does

### PR DESCRIPTION
**Pull request by Iamgoofball**
meme description and changelog entry

### CODE REVIEW

**Comment by Iamgoofball** [comment]
let me handle the comments section for you guys

**Comment by Iamgoofball** [comment]
:-1: I used this all the time, and so did a lot of other players.

**Comment by Iamgoofball** [comment]
@iamgoofball Fuck off

**Comment by Iamgoofball** [comment]
:+1: from random players, flood of :-1: reddit downboats on those posts

**Comment by Iamgoofball** [comment]
ma44 discussing cash lemons again

File: code/modules/surgery/gender_reassignment.dm [hunk 1]
```diff
@@ -1,38 +0,0 @@
-/datum/surgery/gender_reassignment
```
Comment (Iamgoofball) [inline code/modules/surgery/gender_reassignment.dm]: coder commenting that I forgot to remove the dme entry

Comment (Iamgoofball) [inline code/modules/surgery/gender_reassignment.dm]: swearing response from myself

**Comment by Iamgoofball** [comment]
wall of text from Luke Cox/ShadowDimentio/DemonFiren/John_Oxford about how this PR is absolutely terrible without actually addressing any points, effectively saying nothing in as many words as possible

**Comment by Iamgoofball** [comment]
oranges closing the PR for a meme with a meme post

**Comment by Iamgoofball** [comment]
re-opened by Kor because design lead

**Comment by Iamgoofball** [comment]
mix of posts about "ugh why was this closed :+1: this pr was good"

**Comment by Iamgoofball** [comment]
posts replying bitching about why it was re-opened

**Comment by Iamgoofball** [comment]
closed by antur for renaming the PR back

**Comment by Iamgoofball** [comment]
a week later I rename it and beg for reopen, which is granted by oranges

**Comment by Iamgoofball** [comment]
it gets speedmerged by kor to the anger of the players

**Comment by optimumtact** [comment]
Why was this speed merged with no discussion and a security tag to prevent players seeing it?

**Comment by Jordie0608** [comment]
For ref a changelog was made https://github.com/tgstation/tgstation/commit/4e9c809689616af25eaa78807e520f975ae4eeb4

**Comment by phil235** [comment]
why??

**Pull request merged**